python/src/trezorlib/benfen.py

```
@@ -0,0 +1,41 @@
+from . import messages
+from .tools import expect
+
+if TYPE_CHECKING:
+    from .client import TrezorClient
+    from .tools import Address
+    from .protobuf import MessageType
+
+
+@expect(messages.BenfenAddress, field="address", ret_type=str)
+def get_address(
+    client: "TrezorClient", address_n: "Address", show_display: bool = False
```

Ruff (0.8.2)

Boolean-typed positional argument in function definition

(FBT001)